app/form_models/jobseekers/job_application/details/qualifications/secondary/common_form.rb

```diff
@@ -1,14 +1,50 @@
-class Jobseekers::JobApplication::Details::Qualifications::Secondary::CommonForm < Jobseekers::JobApplication::Details::Qualifications::QualificationForm
-  validates :institution, :year, presence: true
-  validate :subject_and_grade_correspond?
-  validates :year, format: { with: /\A\d{4}\z/.freeze }, if: -> { year.present? }
-
-  def subject_and_grade_correspond?
-    return if subject.present? && grade.present?
-
-    errors.add(:subject, I18n.t("qualification_errors.subject_and_grade_correspond.false"))
-    # Empty string: highlight grade field with red, but rely on the user reading the error on subject.
-    # The resulting empty anchor tag in the error summary correctly links to the field.
-    errors.add(:grade, "")
+module Jobseekers::JobApplication::Details::Qualifications::Secondary
+  class CommonForm < ::Jobseekers::JobApplication::Details::Qualifications::QualificationForm
+    MAXIMUM_NUMBER_OF_RESULTS = 6
+
+    validates :institution, :year, presence: true
+    validates :year, format: { with: /\A\d{4}\z/.freeze }, if: -> { year.present? }
```

Comment:
This probably isn't in the acceptance criteria, but can we check if we should validate the year isn't in the future? (This came up for the QTS year question eventually)

Reply:
👍  Have added a note-to-self!

Reply:
We allow people to list qualifications they have not finished (there is a column or two for this, finished_studying and finsished_studying_details), so they ought to be able to list qualifications that end in a future year (within limits?).

Reply:
Actually, it's not so simple.

When we ask if a qual is finished, they can only enter a year if they said "Yes" - i.e. they have finished.

Some forms do not ask if a qual is finished (secondary quals).

Therefore, on some forms it makes sense to validate that the year is this year or earlier - if it's future, they should have selected 'No'.

<img width="827" alt="Screenshot 2021-05-04 at 17 31 06" src="https://user-images.githubusercontent.com/60350599/117037553-84865980-acfe-11eb-829a-65da8b0d4f19.png">